packages/fluentui/react-northstar/src/components/Slider/Slider.tsx

```diff
@@ -217,14 +217,12 @@ const Slider: React.FC<WithAsProp<SliderProps>> & FluentComponentStaticProps = p
         ...htmlInputProps,
```

Comment:
Should we be using `Input.create` instead of `Box` here? @layershifter @mnajdova @pompomon

Reply:
Nope, as this will create a plain `input` while `Input` has different markup, styles and functionality